prime-router/src/main/kotlin/Submission.kt

```diff
@@ -367,9 +367,11 @@ data class Destination(
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 class SubmissionHistory(
-    @JsonProperty("taskId")
+    @JsonProperty("submissionId")
     val actionId: Long,
+    @JsonProperty("timestamp")
     val createdAt: OffsetDateTime,
+    @JsonProperty("sender")
     val sendingOrg: String,
```

Comment:
in SubmissionsResource.ts, you changed sendingOrg to sendend

Reply:
Good catch- this was a typo!